synapse/tests/test_lib_storm.py

```diff
@@ -390,21 +331,20 @@ class StormTest(s_t_utils.SynTest):
             self.eq(nodes[1].ndef, snode.ndef)
 
             # invalid prop
-            q = 'inet:search:query | scrape -p engine foobarbaz'
-            await self.asyncraises(s_exc.BadOptValu, core.nodes(q))
+            #q = 'inet:search:query | scrape -p engine foobarbaz'
```

Comment:
! commented code - why ?

Reply:
because scrape doesn't take property names and/or raise BadOptValu anymore

Reply:
( i'll delete it ) :D